x/validator/keeper/disabled_validator.go

```diff
@@ -0,0 +1,63 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/validator/types"
+)
+
+// SetDisabledValidator set a specific disabledValidator in the store from its index.
+func (k Keeper) SetDisabledValidator(ctx sdk.Context, disabledValidator types.DisabledValidator) {
```

Comment:
A call of this method and the node admin account revocation should be added to `validatorkeeper.Keeper.HandleDoubleSign` method.